man/man1/mash-account-add-azure.1

```diff
@@ -1,12 +1,47 @@
+.\" Process this file with
+.\" groff -man -Tascii mash-account-add-azure.1
+.\"
 .TH "MASH ACCOUNT ADD AZURE" "1" "27-Aug-2018" "" "mash account add azure Manual"
 .SH NAME
-mash\-account\-add\-azure \- Add Azure account given the provided args.
+mash\-account\-add\-azure \- Add Azure account.
 .SH SYNOPSIS
 .B mash account add azure
-[OPTIONS] ACCOUNT_NAME CONTAINER_NAME REGION REQUESTING_USER RESOURCE_GROUP STORAGE_ACCOUNT CREDENTIALS_PATH
+[OPTIONS]
 .SH DESCRIPTION
-Add Azure account given the provided args.
+Add an Azure account in the user name space on the \fImash\fP server. See
+\fBmash-account-add(1)\fP for details about user name-spaces. Unless
+otherwise noted options are required.
 .SH OPTIONS
 .TP
-\fB\-\-group\fP TEXT
-Group name to place account in.
\ No newline at end of file
+\fB--name\fP ACCOUNT_NAME
```

Comment:
If something is required why is an option? Options are normally optional. These are all really required arguments.